src/pocketmine/tile/Sign.php

```diff
@@ -64,6 +64,27 @@ public function setText($line1 = "", $line2 = "", $line3 = "", $line4 = ""){
 
```

Comment:
I would propose an API change, where `setText()` accepts `null` by default and only changes the lines if it is not null.